# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/ncc/hotfix.patch
# Copyright (C) 2022 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

Force the compiler to interpret the constant as unsigned int instead of signed
--- ncc-2.8/nccnav/nccnav.C.orig	2022-06-20 12:36:22.507859884 +0200
+++ ncc-2.8/nccnav/nccnav.C	2022-06-20 12:35:07.938853899 +0200
@@ -331,7 +331,7 @@
 
 struct file_chunk {
 	unsigned int start, end;
-} NoChunk = { ~0, ~0 };
+} NoChunk = { ~0u, ~0u };
 
 class lnnode : public intNode
 {
